requirements.txt

```diff
@@ -1,4 +1,4 @@
 boto3~=1.26
 dbt-core~=1.3.2
-pyathena~=2.19
+pyathena~=2.20
```

Comment:
We also need to change it in the `setup.py` file.

Reply:
That is covered in the pr where we bump to dbt-core 1.4